ribbon-eureka/src/test/java/com/netflix/niws/loadbalancer/DefaultNIWSServerListFilterTest.java

```diff
@@ -40,7 +40,7 @@
 public class DefaultNIWSServerListFilterTest {
     @BeforeClass
     public static void init() {
-    	ConfigurationManager.getDeploymentContext().setValue(ContextKey.zone, "us-eAst-1C");
```

Comment:
The old way is totally messed up.  Archaius defaults to ConfigurationBasedDeploymentContext which doesn't actually update configuration when setValue() is called but does initialize from configuration.  So, if someone actually set the system property `@zone` then it will be picked up by ConfigurationBasedDeploymentContext.  Netflix's NetflixConfiguration actually does the right thing in its implementation of DeploymentContext#setValue by also setting `@zone` in config.  

This is a bit of a breaking change for any code relying on `ConfigurationManager.getDeploymentContext().setValue(ContextKey.zone, "us-eAst-1C");` to set the zone, which is something I would hope isn't actually done in practice.